registry/storage/driver/azure/azure.go

```diff
@@ -73,6 +79,9 @@ func FromParameters(parameters map[string]interface{}) (*Driver, error) {
 		realm = azure.DefaultBaseURL
 	}
 
+	if maxIdleConnsParam, ok := parameters[paramMaxIdleConns]; ok {
+		maxIdleConnsToStorage = maxIdleConnsParam.(int)
```

Comment:
This doesn't look safe, do we do this type of direct casting in any other drivers? My expectation would be to do a type switch, if int use it, if string parse it, otherwise error. We seem to do something similar in the s3 drivers.

Reply:
Thanks! I thought to check the type at first. Then I realized that the driver would be configured once when server starts. If it's not the expected type, server will error out and exit, which would be preferred IMHO. Without the driver being configured correctly the registry anyway cannot work. Just checked the s3 driver, which returned explicit error if the type is not expected. I will follow the pattern then.